packages/@okta/vuepress-site/docs/reference/api/groups/index.md

```diff
@@ -250,6 +252,8 @@ curl -v -X GET \
 
 ##### Response example
 
+> **Note:** <ApiLifecycle access="ea" /> The `source` section and the `source` link in the response example below are only present in groups with type `APP_GROUP`. See [Group attributes](#group-attributes) and [Links object](#links-object).
```

Comment:
I have two (conflicting) thoughts:
1. Do we need this here, or can we leave the details about `source` in the "Group object" > "Group attributes" section below? (I think we can leave the details in the section below)
2. If we do want this, do we want to add details like this to other APIs that return groups? (I think no, that would be too verbose)

Reply:
I was talking more about the note here (`> **Note:** <ApiLifecycle access="ea" /> The `source` section...`) - I agree `"source": ...` needs to be added to any `"APP_GROUP"` object in the documentation at GA time.

Reply:
I do feel that it is necessary to add this note to point readers to the bottom of the page about the details of the attribute since it's a new attribute and is in early access stage.